Problem Statement 1:
In each of the following situations, state whether it is a correctly stated hypothesis testing problem and why?
1.𝐻0: 𝜇 = 25, 𝐻1: 𝜇 ≠ 25 
2.𝐻0: 𝜎 > 10, 𝐻1: 𝜎 = 10 
3.𝐻0: 𝑥 = 50, 𝐻1: 𝑥 ≠ 50 
4.𝐻0: 𝑝 = 0.1, 𝐻1: 𝑝 = 0.5 
5.𝐻0: 𝑠 = 30, 𝐻1: 𝑠 > 30

1.Correct, Null hypothesis always have equal to & it is with respect to mean here
2.Not correct, it doesnot denote equals for null hypothesis
3.Not correct, generally we prove for population from sample (correct if x bar is not sample mean but just symbol)
4.Not correct, alternative hypothesis should be not equal to null hypothesis value; here value is different
5.Subjective & not correct, we never consider sample for hypothesis

Problem Statement 2:
The college bookstore tells prospective students that the average cost of its textbooks is Rs. 52 with a standard deviation of Rs. 4.50. A group of smart statistics students thinks that the average cost is higher. To test the bookstore’s claim against their alternative, the students will select a random sample of size 100. Assume that the mean from their random sample is Rs. 52.80. Perform a hypothesis test at the 5% level of significance and state your decision.

In [1]:
import numpy as np
import pandas as pd
from scipy.stats import binom
import scipy.stats as stat
import math

In [2]:
# need to define null & alternative hypothesis
# H0: Average book cost is equal 52
# H1: Average book cost is not equal 52
MeanP =52 # Polupation mean
MeanS = 52.8 #sample mean
sampl =100 #sample size
sigS = 4.5/math.sqrt(sampl) #Std. deviation of sample
z_book = (MeanS - MeanP)/sigS
z_book

1.7777777777777715

In [3]:
# find probability of z_book score on z score table & it's 1.77
zbook_prob=1-0.9616
zbook_prob

0.03839999999999999

which is less than 0.05 so we fail to reject null hypothesis

Problem Statement 3:
A certain chemical pollutant in the Genesee River has been constant for several years with mean μ = 34 ppm (parts per million) and standard deviation σ = 8 ppm. A group of factory representatives whose companies discharge liquids into the river is now claiming that they have lowered the average with improved filtration devices. A group of environmentalists will test to see if this is true at the 1% level of significance. Assume \ that their sample of size 50 gives a mean of 32.5 ppm. Perform a hypothesis test at the 1% level of significance and state your decision.

In [4]:
#Ho: Improved Filteration gives pollutant = 34ppm
#H1: Improved Filteration gives pollutant != 34ppm
Mupoll = 34 #population mean
Sigpoll =8  # population std. dev.
npoll = 50  # sample size
xbarpoll = 32.5 # sample mean

#Cals std. dev of sample
SigpollS = Sigpoll/math.sqrt(npoll)

# Calc Z score
z_poll_score = (Mupoll-xbarpoll)/SigpollS
print(z_poll_score) # z score is 1.33
probPoll = 1-0.9082 # prob at zscore 1.33 is 0.9082
probPoll

1.3258252147247767


0.09179999999999999

The value is 9% (which is greater than 1%) & we can reject null hypothesis. Means filteration improve pollutant in river

Problem Statement 4:
Based on population figures and other general information on the U.S. population, suppose it has been estimated that, on average, a family of four in the U.S. spends about $1135 annually on dental expenditures. Suppose further that a regional dental association wants to test to determine if this figure is accurate for their area of country. To test this, 22 families of 4 are randomly selected from the population in that area of the country and a log is kept of the family’s dental expenditure for one year. The resulting data are given below. Assuming, that dental expenditure is normally distributed in the population, use the data and an alpha of 0.5 to test the dental association’s hypothesis.
1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994

In [5]:
# Ho: Family of 4 spends on dental =1135
# H1: Family of 4 spends on dental !=1135

MuDnt = 1135 # Polulation mean
DntS = [1008, 812, 1117, 1323, 1308, 1415, 831, 1021, 1287, 851, 930, 730, 699, 872, 913, 944, 954, 987, 1695, 995, 1003, 994]
nDnt = 22 # Sample for family of 4

xbarDnt = np.mean(DntS)
SigmaDnt =  np.std(DntS)# (DntS-xbarDnt)**2/(nDnt-1) we will assume sample std. dev as polulation std. dev
SDnt = SigmaDnt/math.sqrt(nDnt)

# t score
t_Dnt = (MuDnt - xbarDnt)/SDnt
print(t_Dnt)

# here sample is <30 so find tscore for df = 21 & for p=0.5 which is 2.08

2.070747228595759


T score is gives confidence & we fail to reject null hypothesis. So average spend for family of 4 are $1135

Problem Statement 5:
In a report prepared by the Economic Research Department of a major bank the Department manager maintains that the average annual family income on Metropolis is $48,432. What do you conclude about the validity of the report if a random sample of 400 families shows and average income of $48,574 with a standard deviation of 2000?

In [6]:
# Ho: Average salary in Metropolis = 48432
# H1: Average salary in Metropolis != 48432
MuSal = 48432   # population mean
xbarSal = 48574 #sample mean
nSal = 400      #sample size
SSal = 2000     #Sample std. dev

# We assume sample std. dev as population std. dev
SigSal = SSal/math.sqrt(nSal)
z_sal = (MuSal - xbarSal)/SigSal
print(z_sal)

#z score for -1.42 is 0.0778

-1.42


If we assume if we assume probability of 5% level of significance then here what we get for null hypothesis is 0.7 & so we reject null hypothesis. Wich means average salary in Metropolis may higher

Problem Statement 6:
Suppose that in past years the average price per square foot for warehouses in the United States has been $32.28. A national real estate investor wants to determine whether that figure has changed now. The investor hires a researcher who randomly samples 19 warehouses that are for sale across the United States and finds that the mean price per square foot is $31.67, with a standard deviation of $1.29. assume that the prices of warehouse footage are normally distributed in population. If the researcher uses a 5% level of significance, what statistical conclusion can be reached? What are the hypotheses?

In [7]:
#Ho : Avg per sq-ft price of ware house = 32.28
#H1 : Avg per sq-ft price of ware house != 32.28
MuWr = 32.28
xbarWr = 31.67
nWr = 19
Swr = 1.29

# We assume sample std. dev as population std. dev

t_Wr = (MuWr - xbarWr)/(Swr/math.sqrt(nWr))
print(t_Wr)

# tscore of 2.06 for df=18 is b/w confidence level of 0.1 & 0.05 which is > 0.05

2.06118477175179


we reject null hypothesis so alternet is correct, so price per-sqft for warehouse are 31.67

Problem Statement 7: Rahul said calrify with team
Fill in the blank spaces in the table and draw your conclusions from it. 

i) Acceptance region 48.5 < x < 51.5

In [8]:
# Calculate Beta at Mu1 = 52
n1 = 10
Sig =2.5
Mu1 =52

p11 = (48.5 - Mu1)/(Sig/math.sqrt(n1))
print(p11)
p12 = (51.5 - Mu1)/(Sig/math.sqrt(n1))
print(p12)

# As p12 > p11, so our z score lies in between these two p12 < z < p11.
# Find probability at these z score
P11 = 0
P12 = 0.2643
# now Beta = p12 - p11
Beta11 = P12 - P11
Beta11

-4.427188724235731
-0.6324555320336759


0.2643

In [9]:
# Calculate Beta at Mu2 = 50.5
n1 = 10
Sig =2.5
Mu2 =50.5

p13 = (48.5 - Mu2)/(Sig/math.sqrt(n1))
print(p13)
p14 = (51.5 - Mu2)/(Sig/math.sqrt(n1))
print(p14)

# so our z score lies in between these two p13 < z < p14.
# Find probability at these z score
P13 = 0.0057
P14 = 0.8962
# now Beta = p13 +(1 - p14)
Beta12 = P13 +(1- P14)
Beta12

-2.5298221281347035
1.2649110640673518


0.1095

ii) 48 < x < 51

In [10]:
# Calculate Beta at Mu1 = 52
n2 = 10
Sig =2.5
Mu1 =52

p21 = (48.0 - Mu1)/(Sig/math.sqrt(n2))
print(p21)
p22 = (51.0 - Mu1)/(Sig/math.sqrt(n2))
print(p22)

# As p22 > p21, so our z score lies in between these two p22 < z < p21.
# Find probability at these z score
P21 = 0
P22 = 0.1038
# now Beta = p22 - p21
Beta21 = P22 - P21
Beta21

-5.059644256269407
-1.2649110640673518


0.1038

In [11]:
# Calculate Beta at Mu2 = 50.5
n2 = 10
Sig =2.5
Mu2 =50.5

p23 = (48 - Mu2)/(Sig/math.sqrt(n2))
print(p23)
p24 = (51 - Mu2)/(Sig/math.sqrt(n2))
print(p24)

# so our z score lies in between these two p13 < z < p14.
# Find probability at these z score
P23 = 0.0008
P24 = 0.7357
# now Beta = p13 +(1 - p14)
Beta22 = P23 +(1- P24)
Beta22

-3.1622776601683795
0.6324555320336759


0.2651

iii) 48.81 < x < 51.9

In [12]:
# Calculate Beta at Mu1 = 52
n3 = 16
Sig =2.5
Mu1 =52

p31 = (48.81 - Mu1)/(Sig/math.sqrt(n3))
print(p31)
p32 = (51.9 - Mu1)/(Sig/math.sqrt(n3))
print(p32)

# so our z score lies in between these two p31 < z < p32.
# Find probability at these z score
P31 = 0.4364
P32 = 0
# now Beta = p31 - p32
Beta31 = P31 - P32
Beta31

-5.1039999999999965
-0.16000000000000228


0.4364

In [13]:
# Calculate Beta at Mu2 = 50.5
n3 = 16
Sig =2.5
Mu2 =50.5

p33 = (48.81 - Mu2)/(Sig/math.sqrt(n3))
print(p33)
p34 = (51.9 - Mu2)/(Sig/math.sqrt(n3))
print(p34)

# so our z score lies in between these two p33 < z < 1- p34.
# Find probability at these z score
P33 = 0.0032
P34 = 0.9875
# now Beta = P33 + 1- P34
Beta32 = P33 +1 - P34
Beta32

-2.703999999999996
2.2399999999999975


0.015700000000000047

iv) 48.42 <x < 51.58

In [15]:
# Calculate Beta at Mu1 = 52
n4 = 16
Sig =2.5
Mu1 =52

p41 = (48.42 - Mu1)/(Sig/math.sqrt(n4))
print(p41)
p42 = (51.58 - Mu1)/(Sig/math.sqrt(n4))
print(p42)

# As P42> P41, so our z score lies in between these two p42 < z < p41.
# Find probability at these z score
P41 = 0.0
P42 = 0.2514
# now Beta = p31 - p32
Beta41 = P42 - P41
Beta41

-5.727999999999997
-0.6720000000000027


0.2514

In [16]:
# Calculate Beta at Mu2 = 50.5
n4 = 16
Sig =2.5
Mu2 =50.5

p43 = (48.42 - Mu2)/(Sig/math.sqrt(n4))
print(p33)
p44 = (51.58 - Mu2)/(Sig/math.sqrt(n4))
print(p34)

# so our z score lies in between these two p33 < z < 1- p34.
# Find probability at these z score
P43 = 0.0035
P44 = 0.9875
# now Beta = P43 + 1- P44
Beta42 = P43 +(1 - P44)
Beta42

-2.703999999999996
2.2399999999999975


0.015999999999999955

Problem Statement 8:
Find the t-score for a sample size of 16 taken from a population with mean 10 when the sample mean is 12 and the sample standard deviation is 1.5.

In [17]:
nSmpl = 16
Mupop = 10
x_bar_pop = 12
s_pop = 1.5

StdPop = s_pop/math.sqrt(nSmpl) # assume sample std. dev as population std. dev
t_Score_pop = (Mupop - x_bar_pop)/StdPop

print(t_Score_pop)

-5.333333333333333


Problem Statement 9:
Find the t-score below which we can expect 99% of sample means will fall if samples of size 16 are taken from a normally distributed population.

In [18]:
#t score for Confidense level 99% need to be found
df = 15  # degreed of freedonw n-1

# find t-score for df=15 & confidence 99%
t_99_conf = + - 2.603

Problem Statement 10:
If a random sample of size 25 drawn from a normal population gives a mean of 60 and a standard deviation of 4, find the range of t-scores where we can expect to find the middle 95% of all sample means. Compute the probability that (−𝑡0.05 <𝑡<𝑡0.10).

In [19]:
ns=25
x_bar_s=60
s_s = 4

# find df & then value for that df & confidence point for 95%
df =24

t_S_score = + - 1.711

Problem Statement 11:
Two-tailed test for difference between two population means:
Is there evidence to conclude that the number of people travelling from Bangalore to Chennai is different from the number of people travelling from Bangalore to Hosur in a week, given the following:
Population 1: Bangalore to Chennai
n1 = 1200
x1 = 452
s1 = 212
Population 2: Bangalore to Hosur
n2 = 800
x2 = 523
s2 = 185

In [20]:
#Ho: Number of people travel are equal (x1-x2=0)
#H1: Number of people travel are not equal & x1 is higher (x1-x2 >0)  
n1=1200
n2=800
x1=452
x2=523
s1=212
s2=185
import scipy.stats as stat

DiffX=x1-x2
print("Mean Differece = ", DiffX)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume two tail-t test z score for that is 1.65
z_critc = 1.65 # Distance from Mu to 95% confidence interval

Sigx1_x2 = math.sqrt((s1**2/n1)+(s2**2/n2))
Dist_x1x2 = z_critc*Sigx1_x2
print("Distance from Meanfor 95% confidence = ",Dist_x1x2)

Mean Differece =  -71
Distance from Meanfor 95% confidence =  14.77967026442065


95% confidence interval distance of 14.78 from Mu. But the value of Mean difference we got is -71 which lies much far from Mu. So it's in area <0.05% confidence. So we can reject the null hypothesis & accept H1. so people travel Bangalore to Chennai are not equal

Problem Statement 12:¶ 
Is there evidence to conclude that the number of people preferring Duracell battery is different from the number of people preferring Energizer battery, given the following: Population 1: Duracell
n1 = 100
x1 = 308
s1 = 84
Population 2: Energizer
n2 = 100
x2 = 254
s2 = 67

In [21]:
#Ho: Number of people prefer Duracell & Energizer are equal (x1-x2=0)
#H1: Number of people prefer Duracell & Energizer are not equal & x1 is higher (x1-x2 >0)  
nc1=100
nc2=100
xc1=308
xc2=254
sc1=84
sc2=67

DiffXc=xc1-xc2
print("Mean Differece = ",DiffXc)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume two tail-t test z score for that is 1.65
z_Ccritc = 1.65 # Distance from Mu to 95% confidence interval

Sigxc1_xc2 = math.sqrt((sc2**2/nc2)+(sc1**2/nc1))
Dist_cx1x2 = z_Ccritc*Sigxc1_xc2
print("Dist from mean for 95% confidence = ",Dist_cx1x2)

Mean Differece =  54
Dist from mean for 95% confidence =  17.72886417681629


95% confidsence interval distance of 2.03 from Mu. But the value of Mean difference we got is 54 which lies much far from Mu. So it's in area <5% confidence. So we can reject the null hypothesis & accept H1.

Problem Statement 13:
Pooled estimate of the population variance Does the data provide sufficient evidence to conclude that average percentage increase in the price of sugar differs when it is sold at two different prices?
Population 1: Price of sugar = Rs. 27.50
n1 = 14 x1 = 0.317% s1 = 0.12%
Population 2: Price of sugar = Rs. 20.00
n2 = 9 x2 = 0.21% s2 = 0.11%

In [22]:
#Ho: Sugar price % increase is zero (x1-x2=0)
#H1: Sugar price % increase is not zero (x1-x2 >0)  
ns1=14
ns2=9
xs1=0.317
xs2=0.21
ss1=0.12
ss2=0.11

DiffXs=xs1-xs2
print("Mean Differece = ",DiffXs)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume two tail-t test z score for that is 1.65
z_Scritc = 1.65 # Distance from Mu to 95% confidence interval

Sigxs1_xs2 = math.sqrt((ss2**2/ns2)+(ss1**2/ns1))
Dist_sx1x2 = z_Scritc*Sigxs1_xs2
print("Distacne from mean for 95% confidence = ",Dist_sx1x2)

Mean Differece =  0.10700000000000001
Distacne from mean for 95% confidence =  0.08037745774958122


95% confidsence interval distance of 0.08 from Mu. But the value of Mean difference we got is 0.11 which higher. So it's in area of 5% confidence. So we can reject the null hypothesis & so average percentage increase in the price of sugar has effect when it sold at two different prices.

Problem Statement 14: The manufacturers of compact disk players want to test whether a small price reduction is enough to increase sales of their product. Is there evidence that the small price reduction is enough to increase sales of compact disk players?
Population 1: Before reduction
n1 = 15 x1 = Rs. 6598 s1 = Rs. 844
Population 2: After reduction
n2 = 12 x2 = RS. 6870 s2 = Rs. 669

In [23]:
#Ho: Sales difference due to price change is zero (x1-x2=0)
#H1: Sales difference due to price change is not zero (x1-x2 >0)  
npl1=15
npl2=12
xpl1=6598
xpl2=6870
spl1=844
spl2=669

DiffXpl=xpl1-xpl2
print("Mean Differece = ",DiffXpl)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume two tail-t test z score for that is 1.65
z_Plcritc = 1.65 # Distance from Mu to 95% confidence interval

Sigxpl1_xpl2 = math.sqrt((spl2**2/npl2)+(spl1**2/npl1))
Dist_plx1x2 = z_Plcritc*Sigxpl1_xpl2
print("Distacne from mean for 95% confidence = ",Dist_plx1x2)

Mean Differece =  -272
Distacne from mean for 95% confidence =  480.44706875471724


95% confidence interval distance of 480 from Mu. But the value of Mean difference we got is -272 which lower & within 95% area. So we failed to reject the null hypothesis & so compact disc prise difference have no effect on sale of disc player.

Problem Statement 15:
Comparisons of two population proportions when the hypothesized difference is zero. Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980 and 1995.
Population 1: 1980
n1 = 1000 x1 = 53 𝑝 1 = 0.53
Population 2: 1985
n2 = 100 x2 = 43 𝑝 2= 0.53

In [24]:
#Ho: Car loan market share is equal (x1-x2=0)
#H1: Car loan market share is not equal (x1-x2 >0)  
nl1=1000
nl2=100
xl1=53
xl2=53
sl1=0.53
sl2=0.53

DiffXl=xl1-xl2
print("Mean Differece = ",DiffXl)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume two tail-t test z score for that is 1.65
z_Llcritc = 1.65 # Distance from Mu to 95% confidence interval

Sigxl1_xl2 = math.sqrt((sl2**2/nl2)+(sl1**2/nl1))
Dist_lx1x2 = z_Llcritc*Sigxl1_xl2
print("Distacne from mean for 95% confidence = ",Dist_lx1x2)

Mean Differece =  0
Distacne from mean for 95% confidence =  0.09171833377247976


95% confidence interval distance of 0.09 from Mu. But the value of Mean difference we got is 0 which lower & within 95% area. So we failed to reject the null hypothesis & so car loan share of bank is not equal.

Problem Statement 16:
Carry out a one-tailed test to determine whether the population proportion of traveler’s check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as at least 10% higher than the proportion of such buyers when no sweepstakes are on.
Population 1: With sweepstakes
n1 = 300 x1 = 120 𝑝 = 0.40
Population 2: No sweepstakes
n2 = 700 x2 = 140 𝑝 2= 0.20

In [25]:
#Ho: Checks buyer are equal respective of sweepstakes (x1-x2=0)
#H1: Checks buyer are not equal respective of sweepstakes (x1-x2 >0)  
nsw1=300
nsw2=700
xsw1=120
xsw2=140
ssw1=0.4
ssw2=0.2

DiffXsw=xsw1-xsw2
print("Mean Differece = ",DiffXsw)
#assume significance level is 0.05% & find z critical for 95% confidence
# assume one tail-t test z score for that is 1.65
z_Swcritc = 1.65 # Distance from Mu to 95% confidence interval

Sigxsw1_xsw2 = math.sqrt((ssw2**2/nsw2)+(ssw1**2/nsw1))
Dist_Swx1x2 = z_Swcritc*Sigxsw1_xsw2
print("Distacne from mean for 95% confidence = ",Dist_Swx1x2)

Mean Differece =  -20
Distacne from mean for 95% confidence =  0.04009453115540109


95% confidence interval distance of 0.04 from Mu. But the value of Mean difference we got is -20 which is within 5% area. So we can reject the null hypothesis & so sweepstake have impact on but with checks when sweepstakes is on.

Problem Statement 17:
A die is thrown 132 times with the following results:
Number turned up: 1, 2, 3, 4, 5, 6 Frequency: 16, 20, 25, 14, 29, 28 Is the die unbiased? Consider the degrees of freedom as 𝑝=1.

In [26]:
# Probability to get a number on fair die is 1/6
# We will find probability of numbers as with frequency

Pb1 = 16*1/6 # Probability of getting 1
Pb2 = 20*1/6 # Probability of getting 1
Pb3 = 25*1/6 # Probability of getting 1
Pb4 = 14*1/6 # Probability of getting 1
Pb5 = 29*1/6 # Probability of getting 1
Pb6 = 28*1/6 # Probability of getting 1

print(Pb1, Pb2,Pb3,Pb4,Pb5,Pb6)

# Assume confidence level is 95% & significance level is 5%

2.6666666666666665 3.3333333333333335 4.166666666666667 2.3333333333333335 4.833333333333333 4.666666666666667


The probability of the number after frequency is lower than 5% so it's fair die

Problem Statement 18:
In a certain town, there are about one million eligible voters. A simple random sample of 10,000 eligible voters was chosen to study the relationship between gender and participation in the last election. The results are summarized in the following 2X2 (read two by two) contingency table: 
We would want to check whether being a man or a woman (columns) is independent of having voted in the last election (rows). In other words, is “gender and voting independent”?

In [28]:
#Ho: Gender & voting are independent
#H1: Gender & voting are dependent

#Assume voting is independent of gender & find Expected value for it
TotVoted = 2792+3591
TotNVoted = 1486+2131
TotMen = 2792+1486
TotWMen = 3591+2131

#Exp Value = (row total * column total)/ Grand total
ExpMenVote = (TotVoted* TotMen)/10000
ExpWMenVote = (TotVoted* TotWMen)/10000
ExpMenNVote = (TotNVoted* TotMen)/10000
ExpWMenNVote = (TotNVoted* TotWMen)/10000
print(ExpMenNVote)
print(ExpWMenNVote)
print(ExpMenVote)
print(ExpWMenVote)

GenderVoteChiSqr = (2792-ExpMenVote)**2/ExpMenVote + (3591 -  ExpWMenVote)**2/ExpWMenVote + (1486 - ExpMenNVote)**2/ExpMenNVote + (2131 - ExpWMenNVote)**2/ExpWMenNVote
GenderVoteChiSqr

1547.3526
2069.6474
2730.6474
3652.3526


6.660455899328067

Base on porbability of ChiSquare null hypothesis, we can reject null hypothesis. So voting are dependent of gender

Problem Statement 19:
A sample of 100 voters are asked which of four candidates they would vote for in an election. The number supporting each candidate is given below:  Do the data suggest that all candidates are equally popular? [Chi-Square = 14.96, with 3 df, 𝑝 0.05 

In [29]:
#Ho: Candidates are equally popular
#H1: Candidates are not equally popular
ExpVal = 100/4
Hug = 41
Rear = 19
White = 24
Char = 16
ChiSqrPopularity = (Hug-ExpVal)**2/ExpVal +(Rear-ExpVal)**2/ExpVal +(White-ExpVal)**2/ExpVal + (Char-ExpVal)**2/ExpVal
ChiSqrPopularity

14.959999999999999

As ChiSqr value greater than 5%, we reject null hypothesis. So Candidates don't have equal popularity

Problem Statement 20:
Children of three ages are asked to indicate their preference for three photographs of adults. Do the data suggest that there is a significant relationship between age and photograph preference? What is wrong with this study? [Chi-Square = 29.6, with 4 df: 𝑝 < 0.05]

In [30]:
#Ho: Child Age & adult photograph preference has no relation
#H1: Child Age & adult photograph preference has relation
Child5_6 = 18+22+20
PhotoA = 18+2+20
PhotoB = 22+28+10
Child7_8 = 2+28+40
PhotoC = 20+40+40
Child9_10 = 20+10+40
GrdTotal = 200
#Exp Value = (row total * column total)/ Grand total
ExpChid1 = (Child5_6*PhotoA)/GrdTotal
ExpChid2 = (Child5_6*PhotoB)/GrdTotal
ExpChid3 = (Child5_6*PhotoC)/GrdTotal
ExpChid4 = (Child7_8*PhotoA)/GrdTotal
ExpChid5 = (Child7_8*PhotoB)/GrdTotal
ExpChid6 = (Child7_8*PhotoC)/GrdTotal
ExpChid7 = (Child9_10*PhotoA)/GrdTotal
ExpChid8 = (Child9_10*PhotoB)/GrdTotal
ExpChid9 = (Child9_10*PhotoC)/GrdTotal


ChiSqr56 = (18-ExpChid1)**2/ExpChid1 + (22-ExpChid2)**2/ExpChid2+ (20-ExpChid3)**2/ExpChid3
ChiSqr78 = (2-ExpChid4)**2/ExpChid4 + (28-ExpChid5)**2/ExpChid5+ (40-ExpChid6)**2/ExpChid6
ChiSqr910 = (20-ExpChid7)**2/ExpChid7 + (10-ExpChid8)**2/ExpChid8+ (40-ExpChid9)**2/ExpChid9

ChiSqrChld = ChiSqr56 + ChiSqr78+ ChiSqr910
ChiSqrChld

29.603174603174605

As chi sqr value >5% we reject null hypothesis and Child Age & adult photograph preference has relation.
what's wrong: On age 7-8 & 9-10 one of sample have less random sample than expected

Problem Statement 21:
A study of conformity using the Asch paradigm involved two conditions: one where one confederate supported the true judgement and another where no confederate gave the correct response.  Is there a significant difference between the "support" and "no support" conditions in the frequency with which individuals are likely to conform? [Chi-Square = 19.87, with 1 df: 𝑝 < 0.05].

In [31]:
#Ho: It confirms support
#H1: It confirms no support
TotCon = 18+40
TotNCon = 32+10
TotSup = 18+32
TotNSup = 40+10
TotGr = 100
#Exp Value = (row total * column total)/ Grand total
Exp1 = (TotCon*TotSup)/TotGr
Exp2 = (TotCon*TotNSup)/TotGr
Exp3 = (TotNCon*TotSup)/TotGr
Exp4 = (TotNCon*TotNSup)/TotGr

ChiSqrSupp = (18-Exp1)**2/Exp1 + (40-Exp2)**2/Exp2+ +(32-Exp3)**2/Exp3 + (10-Exp4)**2/Exp4
ChiSqrSupp

19.868637110016422

Problem Statement 22:
We want to test whether short people differ with respect to their leadership qualities (Genghis Khan, Adolf Hitler and Napoleon were all stature-deprived, and how many midget MP's are there?) The following table shows the frequencies with which 43 short people and 52 tall people were categorized as "leaders", "followers" or as "unclassifiable". Is there a relationship between height and leadership qualities?
[Chi-Square = 10.71, with 2 df: 𝑝 < 0.01]. 

In [32]:
#Ho: Height & leadership have no connection
#H1: Height & leadership have connection
TotShort = 12+22+9
TotTall = 32+14+6
TotLead = 12+32
TotFoll = 22+14
TotUnclass = 9+6
TotSamp = 95

#Exp Value = (row total * column total)/ Grand total
ExpH1 = (TotShort*TotLead)/TotSamp
ExpH2 = (TotTall*TotLead)/TotSamp
ExpH3 = (TotShort*TotFoll)/TotSamp
ExpH4 = (TotTall*TotFoll)/TotSamp
ExpH5 = (TotShort*TotUnclass)/TotSamp
ExpH6 = (TotTall*TotUnclass)/TotSamp


ChiSqrLead = (12-ExpH1)**2/ExpH1 +(32-ExpH2)**2/ExpH2 +(22-ExpH3)**2/ExpH3 + (14-ExpH4)**2/ExpH4+ (9-ExpH5)**2/ExpH5+(6-ExpH6)**2/ExpH6
ChiSqrLead

10.712198008709638

ChiSqr value higher than 0.01 or 1% so we reject null hypothesis. And so there is relationship b/w height & leadership

Problem Statement 23:
Each respondent in the Current Population Survey of March 1993 was classified as employed, unemployed, or outside the labor force. The results for men in California age 35-44 can be cross-tabulated by marital status, as follows: 
Men of different marital status seem to have different distributions of labor force status. Or is this just chance variation? (you may assume the table results from a simple random sample.)

In [33]:
#Ho: Job & marital status have no relation
#H1: Job & marital status have no relation

TotMarr = 679+63+42
TotWDS = 103+10+18
TotNMarr = 114+20+25
TotEmp = 679+103+114
TotUEmp = 63+10+20
TotLBF = 42+18+25
TotJ = TotMarr+TotWDS +TotNMarr

#Exp Value = (row total * column total)/ Grand total
ExpJ1 = (TotMarr*TotEmp)/TotJ
ExpJ2 = (TotWDS*TotEmp)/TotJ
ExpJ3 = (TotNMarr*TotEmp)/TotJ
ExpJ4 = (TotMarr*TotUEmp)/TotJ
ExpJ5 = (TotWDS*TotUEmp)/TotJ
ExpJ6 = (TotNMarr*TotUEmp)/TotJ
ExpJ7 = (TotMarr*TotLBF)/TotJ
ExpJ8 = (TotWDS*TotLBF)/TotJ
ExpJ9 = (TotNMarr*TotLBF)/TotJ

ChisqrEmp = (679-ExpJ1)**2/ExpJ1 + (103-ExpJ2)**2/ExpJ2 + (114-ExpJ3)**2/ExpJ3
ChisqrUEmp = (63-ExpJ4)**2/ExpJ4 + (10-ExpJ5)**2/ExpJ5 + (20-ExpJ6)**2/ExpJ6
ChisqrLBF = (42-ExpJ7)**2/ExpJ7 + (18-ExpJ8)**2/ExpJ8 + (25-ExpJ9)**2/ExpJ9
ChiSqrJob = ChisqrEmp+ChisqrUEmp+ChisqrLBF

ChiSqrJob

31.613103194077983

For significance level p<0.01 , df = 4 (3-1 * 3-1 ) value of chi sqr is much greater, so we reject null hypothesis. SO there is relation between job & marital status